Ballerina.toml

```diff
@@ -2,6 +2,6 @@
 org-name= "ballerinax"
 license= ["Apache-2.0"]
 authors = ["Ballerina"]
-version= "0.11.0"
-keywords = ["amazonsqs", "Amazon", "SQS", "Ballerina", "connector", "client"]
-repository = "https://github.com/ballerina-platform/module-amazonsqs"
+version= "0.12.0"
```

Comment:
the version number should be 0.99.0 for the preview release

Reply:
Changed.